### PR TITLE
Feature/data processing unit implementation

### PR DESCRIPTION
**Pull request by jacekdebski**

### CODE REVIEW

File: src/components/data_processing_unit/data_processing_unit.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        std_dev: float = self.__calculate_standard_deviation(mid_values)
+        co_var: float = self.__calculate_coefficient_of_variation(mid_values)
+
+        if any(result is None for result in [median, mode, std_dev, co_var]):
+            status = "Some statistics values may have been calculated incorrectly."
+        else:
+            status = "All statistical values have been calculated correctly."
+
+        result_statistics = {
+            "median": median,
```
Comment (mikoogla) [inline src/components/data_processing_unit/data_processing_unit.py:63]: median is imported from statistics and also median: float declaration there is.

**Review by mikoogla — CHANGES_REQUESTED** [review]
name fix to do

File: src/components/data_processing_unit/data_processing_unit.py (hunk 1 above)
Comment (jacekdebski) [reply src/components/data_processing_unit/data_processing_unit.py:63]: imported all functions from statistics module to easier differentiate local variables

**Review by rfilipczak — DISMISSED** [review]
lgtm

**Review by rfilipczak — COMMENTED** [review]
nvm

**Review by rfilipczak — APPROVED** [review]

**Review by mikoogla — APPROVED** [review]

**Pull request merged**